nyc_taxis/operations/default.json

```diff
@@ -1019,32 +1044,32 @@
     {
       "name": "stats_count_group_by_esql",
       "operation-type": "esql",
-      "query" : "FROM nyc_taxis METADATA _id | stats count(passenger_count) by _id | LIMIT 1000"
+      "query" : "FROM nyc_taxis_sample METADATA _id | stats count(passenger_count) by _id"
```

Comment:
Note the `LIMIT 1000` was moved after the queries in this other pr: https://github.com/elastic/rally-tracks/pull/873

Reply:
Having the limit before or after the stats completely changes the behaviour of the query. The other PR moved it after, and this one effectively moves it before again. I'm assuming we will see performance differences with both changes?

Reply:
Having said that, I understand the purpose of these queries was to limit before, so the change in this current PR is moving back towards that original purpose.